tests/model_registry/rbac/test_mr_rbac.py

```diff
@@ -0,0 +1,299 @@
+
+        Args:
+            model_registry_instance: The Model Registry instance to test against
+            model_registry_namespace: The namespace where Model Registry is deployed
+            admin_client: The admin client for accessing the cluster
+            test_idp_user_session: The test user session containing both admin and user contexts
+            use_admin_context: Whether to use admin context (True) or user context (False)
+
+        Raises:
+            AssertionError: If access permissions don't match expectations
+            ForbiddenException: Expected for normal users, unexpected for admin users
+        """
+        assert model_registry_instance.name == MR_INSTANCE_NAME
+
+        context_to_use = (
+            test_idp_user_session.original_context if use_admin_context else test_idp_user_session.user_context
+        )
+        with use_context(context_to_use):
+            _, client_args = setup_mr_client(
+                model_registry_instance=model_registry_instance,
+                model_registry_namespace=model_registry_namespace,
+                admin_client=admin_client,
+            )
+
+            if use_admin_context:
+                mr_client = ModelRegistryClient(**client_args)
+                assert mr_client is not None, "Client initialization failed for admin user"
+                LOGGER.info("Successfully created Model Registry client for admin user")
+            else:
+                with pytest.raises(ForbiddenException) as exc_info:
+                    _ = ModelRegistryClient(**client_args)
+                assert exc_info.value.status == 403, f"Expected HTTP 403 Forbidden, but got {exc_info.value.status}"
+                LOGGER.info("Successfully received expected HTTP 403 status code")
+
+    @pytest.mark.sanity
+    def test_user_added_to_group(
+        self: Self,
+        model_registry_instance: ModelRegistry,
+        model_registry_namespace: str,
+        admin_client: DynamicClient,
+        test_idp_user_session: TestUserSession,
+    ):
+        """
+        Test that a user's access to Model Registry changes when added to a group.
+
+        This test verifies that:
+        1. A normal user cannot access the Model Registry initially
+        2. After adding the user to the appropriate group, they gain access
+        3. After removing the user from the group, they lose access
+
+        Args:
+            model_registry_instance: The Model Registry instance to test against
+            model_registry_namespace: The namespace where Model Registry is deployed
+            admin_client: The admin client for accessing the cluster
+            test_idp_user_session: The test user session containing both admin and user contexts
+
+        Raises:
+            AssertionError: If access permissions don't match expectations
+            ForbiddenException: Expected before group addition, unexpected after
+        """
+        LOGGER.info(
```

Comment:
We already print start of tests, that would tell you which test is running. Do you still need it?

Reply:
I think it is better to have this, it is helping in recognize faster the test and also it is targeting the test and not setup or teardown
```-----Test ..... -----```

Reply:
We print ```--------------<test name>------------``` at the begining, then `-----SETUP-------` for fixtures, `---------CALL---------------` for test body and then `-------------TEARDOWN----------`. Is not that good enough?

Reply:
<img width="2415" alt="Screenshot 2025-05-20 at 13 57 23" src="https://github.com/user-attachments/assets/50ab4bc1-1a28-4456-8410-8e0c4e4c9d8f" />
in my opinion that works only if the setup is really short and close to `----CALL-----` otherwise you do not see on the terminal which test the `CALL` is referring to. Check this screenshot. But if this is blocking the PR I could remove

Reply:
Not going to block the PR, but I don't see why `CALL` message is not enough to understand start of the test. Then may be we should update `CALL` to something meaningful?

Reply:
maybe we should add the test name ora something more in `CALL` for how is it now I found difficult to know which test is been called, especially if the `SETUP` is verbose. But I would like to not do the change in this PR